tests/app/questionnaire/test_answer_store_updater.py

```diff
@@ -0,0 +1,249 @@
+# import unittest
```

Comment:
I was debating whether I should keep this as well?. When modifying these to work with the current version of master, I seemed to end up mocking a lot on top of what's already mocked and seemed almost pointless.

The scenarios are covered via integration tests for the questionnaire.py